spatialmath/quaternion.go

```diff
@@ -101,7 +101,7 @@ func QuatToOV(q quat.Number) *OrientationVector {
 
 	// The contents of ov.newX.Kmag are not in radians but we can use angleEpsilon anyway to check how close we are to
 	// the pole because it's a convenient small number
-	if 1-math.Abs(newZ.Kmag) > defaultAngleEpsilon {
+	if 1-math.Abs(newZ.Kmag) > OrientationVectorPoleRadius {
```

Comment:
Do these two lines require updating tests?

Reply:
defaultAngleEpsilon is equal to the new introduced variable, this PR (and recent discussions around [RSDK-5417](https://viam.atlassian.net/browse/RSDK-5417)  ) made me realize that this should probably be a specific named thing.